src/main.js

```diff
@@ -15,8 +16,36 @@ var currentCover ;
 
 // Add your event listeners here 👇
 randomizeButton.addEventListener('click', randomizeCovers);
-
+userCoverButton.addEventListener('click', userCoverPage);
+viewSavedButton.addEventListener('click', viewSavedPage);
+homeButton.addEventListener('click', goHome);
 // Create your event handlers and other functions here 👇
+function userCoverPage() {
+mainCover.classList.add('hidden');
+formSection.classList.remove('hidden');
+randomizeButton.classList.add('hidden');
+saveCoverButton.classList.add('hidden');
+homeButton.classList.remove('hidden');
+}
+
+function viewSavedPage() {
+  mainCover.classList.add('hidden');
+  savedCoversPage.classList.remove('hidden');
+  randomizeButton.classList.add('hidden');
+  // viewSavedButton.classList.add('hidden');
```

Comment:
Let's delete this line while learning how to resolve a comment which prevents the PR from being merged.